### ISSUE TITLE
sitemap.txt: no such file

### ISSUE THREAD

**Issue by love4taylor** [hexojs/hexo-generator-sitemap#139]
```
sitemap:
  path:
    - sitemap.xml
    - sitemap.txt
```

```
❯ hexo s
err: Error: ENOENT: no such file or directory, open '/Users/love4taylor/Git/blog/node_modules/hexo-generator-sitemap/sitemap.txt'
```

```
❯ grep version /Users/love4taylor/Git/blog/node_modules/hexo-generator-sitemap/package.json
  "version": "3.0.0",

❯ ll /Users/love4taylor/Git/blog/node_modules/hexo-generator-sitemap/           
total 40
-rw-r--r--  1 love4taylor  staff   1.0K  3 11 20:16 LICENSE
-rw-r--r--  1 love4taylor  staff   1.5K  3 11 20:16 README.md
-rw-r--r--  1 love4taylor  staff   526B  3 11 20:16 index.js
drwxr-xr-x  5 love4taylor  staff   160B  3 11 20:20 lib
drwxr-xr-x  3 love4taylor  staff    96B  3 11 20:20 node_modules
-rw-r--r--  1 love4taylor  staff   1.0K  3 11 20:16 package.json
-rw-r--r--  1 love4taylor  staff   1.0K  3 11 20:16 sitemap.xml
```

**Comment by SukkaW** [comment]
@uiolee Can you have a look at this?

We will revert #136 if you can't help with the problem.

**Comment by uiolee** [comment]
@love4taylor @SukkaW ,

sorry that forgot to edit package.json `file:` to include `sitemap.txt`. new commit is pushing.

before publish it to npm, try some ways to use the plugin.(one of them)

1. remove `- sitemap.txt` from yaml to diasble txt generate.
2. manualy download `sitemap.txt` to `/yourblog/` form github, and config yaml `template_txt:`
3. manualy download `sitemap.txt` to `/node_modules/hexo-generator-sitemap/` form github.

**Comment by yoshinorin** [comment]
@love4taylor @SukkaW @uiolee 
I just merged https://github.com/hexojs/hexo-generator-sitemap/pull/141 & published 3.0.1. Now it works well.
Thank you :)
